apps/studio/src/features/editing-experience/components/LayoutSelection.tsx

```diff
@@ -0,0 +1,245 @@
+// Component for layout selection
+
+import React, { useState } from 'react'
+import {
+  Box,
+  Grid,
+  GridItem,
+  HStack,
+  Image,
+  Text,
+  VStack,
+  Icon,
+} from '@chakra-ui/react'
+import { Button, Link } from '@opengovsg/design-system-react'
+import { BiLeftArrowAlt, BiRightArrowAlt, BiShow } from 'react-icons/bi'
+import Preview, {
+  type PreviewProps,
+} from '~/features/editing-experience/components/Preview'
+import articleLayoutPreview from '~/features/editing-experience/data/articleLayoutPreview.json'
+import contentLayoutPreview from '~/features/editing-experience/data/contentLayoutPreview.json'
+
+export interface LayoutSelectionProps {
+  pageName: string
+  // preparing to make the trpc call to create page. required props
+  pageUrl: string
+  siteId: string
+  folderId: string
+}
+
+const LAYOUT_DATA = [
+  {
+    layoutDisplayName: 'Default',
+    layoutTypename: 'content',
+    layoutDescription: 'This is the most basic layout for your content.',
+    imageSource: '/assets/layout-card/default_layout_card.webp',
+    previewJson: contentLayoutPreview,
+  },
+  {
+    layoutDisplayName: 'Article',
+    layoutTypename: 'article',
+    layoutDescription:
+      'Designed for the perfect reading experience. Use this layout for text-heavy content, such as news, press releases, and speeches',
+    imageSource: '/assets/layout-card/article_layout_card.webp',
+    previewJson: articleLayoutPreview,
+  },
+] as const
+
+const dataAttr = (value: unknown) => (!!value ? true : undefined) // Keeping this here for now, might shift out to a utils file when necessary.
```

Comment:
can just shift to utils file now la haha, confirm use somewhere else in the future one